packages/contracts/src/contract-deployment/config.ts

```diff
@@ -254,5 +254,8 @@ export const makeContractDeployConfig = async (
     ERC1820Registry: {
       factory: getContractFactory('ERC1820Registry'),
     },
+    OVM_ProxyEOA: {
+      factory: getContractFactory('OVM_ProxyEOA', undefined, true),
```

Comment:
What are these additional arguments? `undefined` is probably a provider, what is `true`? imo Python kwargs are handy for this kind of thing which can be replicated by making the second argument an object

Reply:
Agree that we should make these additional arguments an object. Although probably deserves its own issue.